src/main/kotlin/org/rust/lang/core/resolve2/CrateDefMap.kt

```diff
@@ -291,6 +294,15 @@ class ModData(
 
     lateinit var asVisItem: VisItem
 
+    var directoryContainedAllChildFiles: FileId = ownedDirectoryId ?: parent!!.directoryContainedAllChildFiles
+
+    val hasPathAttributeRelativeToParentFile: Boolean
```

Comment:
What `RelativeToParentFile` part means? It behaves like "it has a path attribute or any ancestor has a path attribute", right?

Reply:
It means that it have path attribute or any parent inline mod (that is mod which is inside the file containing mod decl) has path attribute